djangoTrading/__init__.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
You do not need `unicode_literals` when writing in Python 3
The same as `absolute_import`